<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="loadedImg">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>200</y>
      <width>259</width>
      <height>259</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:black;
border: 3px solid rgb(72, 169, 166);
	font: 700 20px &quot;Segoe UI&quot;;
color:rgb(72, 169, 166);
}</string>
    </property>
    <property name="text">
     <string>Click here to load image</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="filteredImg">
    <property name="geometry">
     <rect>
      <x>1028</x>
      <y>200</y>
      <width>259</width>
      <height>259</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:black;
border: 3px solid rgb(72, 169, 166);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1366</width>
      <height>768</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="sources.qrc">:/Images/background.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="mergeBtn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>540</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(123, 117, 181); border: none;
color:white;
border-radius: 12px;
	font: 700 20px &quot;Segoe UI&quot;;
}
QPushButton::hover{
background-color: rgb(98, 94, 145); border: none;
}</string>
    </property>
    <property name="text">
     <string>Merge</string>
    </property>
   </widget>
   <widget class="QPushButton" name="invertBtn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>540</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(123, 117, 181); border: none;
color:white;
border-radius: 12px;
	font: 700 20px &quot;Segoe UI&quot;;
}
QPushButton::hover{
background-color: rgb(98, 94, 145); border: none;
}</string>
    </property>
    <property name="text">
     <string>Invert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="flipBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>540</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(123, 117, 181); border: none;
color:white;
border-radius: 12px;
	font: 700 20px &quot;Segoe UI&quot;;
}
QPushButton::hover{
background-color: rgb(98, 94, 145); border: none;
}</string>
    </property>
    <property name="text">
     <string>Flip</string>
    </property>
   </widget>
   <widget class="QPushButton" name="blurBtn">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>540</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(123, 117, 181); border: none;
color:white;
border-radius: 12px;
	font: 700 20px &quot;Segoe UI&quot;;
}
QPushButton::hover{
background-color: rgb(98, 94, 145); border: none;
}</string>
    </property>
    <property name="text">
     <string>Blur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BlackAndWhiteBtn">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>540</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(123, 117, 181); border: none;
color:white;
border-radius: 12px;
	font: 700 18px &quot;Segoe UI&quot;;
}
QPushButton::hover{
background-color: rgb(98, 94, 145); border: none;
}</string>
    </property>
    <property name="text">
     <string>Black and white</string>
    </property>
   </widget>
   <widget class="QPushButton" name="darkenBtn">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>540</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(123, 117, 181); border: none;
color:white;
border-radius: 12px;
	font: 700 20px &quot;Segoe UI&quot;;
}
QPushButton::hover{
background-color: rgb(98, 94, 145); border: none;
}</string>
    </property>
    <property name="text">
     <string>Darken</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lightenBtn">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>540</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(123, 117, 181); border: none;
color:white;
border-radius: 12px;
	font: 700 20px &quot;Segoe UI&quot;;
}
QPushButton::hover{
background-color: rgb(98, 94, 145); border: none;
}</string>
    </property>
    <property name="text">
     <string>Lighten</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadBtn">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>200</y>
      <width>259</width>
      <height>259</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: transparent; border: none;
color:white;
border-radius: 12px;
color:rgb(72, 169, 166);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="flip_v">
    <property name="geometry">
     <rect>
      <x>1028</x>
      <y>200</y>
      <width>256</width>
      <height>128</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color:white;
background-color: rgba(0, 0, 0, 140); border: none;
border-bottom: 3px solid rgb(72, 169, 166);
	font: 700 20px &quot;Segoe UI&quot;;
}

QPushButton::hover{
background-color: rgba(0, 0, 0, 120);
}</string>
    </property>
    <property name="text">
     <string>flip vertically</string>
    </property>
   </widget>
   <widget class="QPushButton" name="flip_h">
    <property name="geometry">
     <rect>
      <x>1028</x>
      <y>328</y>
      <width>256</width>
      <height>128</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color:white;
background-color: rgba(0, 0, 0, 140); border: none;
	font: 700 20px &quot;Segoe UI&quot;;
}

QPushButton::hover{
background-color: rgba(0, 0, 0, 120); border: none;
}</string>
    </property>
    <property name="text">
     <string>flip horizantally</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>620</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(123, 117, 181); border: none;
color:white;
border-radius: 12px;
	font: 700 20px &quot;Segoe UI&quot;;
}
QPushButton::hover{
background-color: rgb(98, 94, 145); border: none;
}</string>
    </property>
    <property name="text">
     <string>Rotate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotate_90">
    <property name="geometry">
     <rect>
      <x>1156</x>
      <y>200</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color:white;
background-color: rgba(0, 0, 0, 140); border: none;
border-bottom: 3px solid rgb(72, 169, 166);
	font: 700 20px &quot;Segoe UI&quot;;
}

QPushButton::hover{
background-color: rgba(0, 0, 0, 120);
}</string>
    </property>
    <property name="text">
     <string>90</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotate_180">
    <property name="geometry">
     <rect>
      <x>1028</x>
      <y>200</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color:white;
background-color: rgba(0, 0, 0, 140); border: none;
border-bottom: 3px solid rgb(72, 169, 166);
border-right: 3px solid rgb(72, 169, 166);
	font: 700 20px &quot;Segoe UI&quot;;
}

QPushButton::hover{
background-color: rgba(0, 0, 0, 120);
}</string>
    </property>
    <property name="text">
     <string>180</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotate_270">
    <property name="geometry">
     <rect>
      <x>1028</x>
      <y>328</y>
      <width>256</width>
      <height>128</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color:white;
background-color: rgba(0, 0, 0, 140); border: none;
	font: 700 20px &quot;Segoe UI&quot;;
}

QPushButton::hover{
background-color: rgba(0, 0, 0, 120);
}</string>
    </property>
    <property name="text">
     <string>270</string>
    </property>
   </widget>
   <widget class="QPushButton" name="detectBtn">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>620</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(123, 117, 181); border: none;
color:white;
border-radius: 12px;
	font: 700 20px &quot;Segoe UI&quot;;
}
QPushButton::hover{
background-color: rgb(98, 94, 145); border: none;
}</string>
    </property>
    <property name="text">
     <string>Detect edges</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enlargeBtn">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>620</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(123, 117, 181); border: none;
color:white;
border-radius: 12px;
	font: 700 20px &quot;Segoe UI&quot;;
}
QPushButton::hover{
background-color: rgb(98, 94, 145); border: none;
}</string>
    </property>
    <property name="text">
     <string>Enlarge</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enlarge_1">
    <property name="geometry">
     <rect>
      <x>1156</x>
      <y>200</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color:white;
background-color: rgba(0, 0, 0, 140); border: none;
border-bottom: 3px solid rgb(72, 169, 166);
	font: 700 20px &quot;Segoe UI&quot;;
}

QPushButton::hover{
background-color: rgba(0, 0, 0, 120);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enlarge_2">
    <property name="geometry">
     <rect>
      <x>1028</x>
      <y>200</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color:white;
background-color: rgba(0, 0, 0, 140); border: none;
border-bottom: 3px solid rgb(72, 169, 166);
border-right: 3px solid rgb(72, 169, 166);
	font: 700 20px &quot;Segoe UI&quot;;
}

QPushButton::hover{
background-color: rgba(0, 0, 0, 120);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enlarge_3">
    <property name="geometry">
     <rect>
      <x>1028</x>
      <y>328</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color:white;
background-color: rgba(0, 0, 0, 140); border: none;
border-right: 3px solid rgb(72, 169, 166);
	font: 700 20px &quot;Segoe UI&quot;;
}

QPushButton::hover{
background-color: rgba(0, 0, 0, 120);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enlarge_4">
    <property name="geometry">
     <rect>
      <x>1156</x>
      <y>328</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color:white;
background-color: rgba(0, 0, 0, 140); border: none;
	font: 700 20px &quot;Segoe UI&quot;;
}

QPushButton::hover{
background-color: rgba(0, 0, 0, 120);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>loadedImg</zorder>
   <zorder>filteredImg</zorder>
   <zorder>mergeBtn</zorder>
   <zorder>invertBtn</zorder>
   <zorder>flipBtn</zorder>
   <zorder>blurBtn</zorder>
   <zorder>BlackAndWhiteBtn</zorder>
   <zorder>darkenBtn</zorder>
   <zorder>lightenBtn</zorder>
   <zorder>loadBtn</zorder>
   <zorder>flip_v</zorder>
   <zorder>flip_h</zorder>
   <zorder>rotateBtn</zorder>
   <zorder>rotate_90</zorder>
   <zorder>rotate_180</zorder>
   <zorder>rotate_270</zorder>
   <zorder>detectBtn</zorder>
   <zorder>enlargeBtn</zorder>
   <zorder>enlarge_1</zorder>
   <zorder>enlarge_2</zorder>
   <zorder>enlarge_3</zorder>
   <zorder>enlarge_4</zorder>
  </widget>
 </widget>
 <resources>
  <include location="sources.qrc"/>
 </resources>
 <connections/>
</ui>
